sdk/iot/core/inc/az_iot_core.h

```diff
@@ -18,6 +18,17 @@
 
 #include <_az_cfg_prefix.h>
 
+/**
+ * @brief Azure IoT service MQTT connection properties.
+ *
+ */
+enum
+{
+  AZ_CLIENT_DEFAULT_MQTT_CONNECT_PORT = 8883,
+  AZ_CLIENT_DEFAULT_MQTT_CONNECT_CLEAN_SESSION = 0x20,
```

Comment:
Clean session is a boolean in most MQTT libraries. I'm not sure about the value which appears to be particular to an MQTT implementation.
(3.1.2.3 Connect Flags of the [MQTT Spec](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html)